app/views/email/CaptureEmailView.scala.html

```diff
@@ -32,15 +33,19 @@ <h1 id="page-heading" class="heading-large">@messages("captureEmail.title")</h1>
 
     @mainTemplate(if(emailForm.errors.nonEmpty) messages("common.error.prefixTitle", messages("captureEmail.title")) else messages("captureEmail.title")) {
 
-        <a class="link-back" href="@appConfig.dynamicJourneyEntryUrl(user.isAgent)">@messages("base.back")</a>
+    @if(letterToConfirmedEmail == true) {
```

Comment:
`if(letterToConfirmedEmail == true)` can be simplified to `if(letterToConfirmedEmail)`